install_config/aggregate_logging.adoc

```diff
@@ -300,6 +300,44 @@ when `openshift_logging_use_ops` is set to `true`.
 |Equivalent to `openshift_logging_kibana_replica_count` for Ops cluster
 when `openshift_logging_use_ops` is set to `true`.
 
+|`openshift_logging_es_allow_external`
+|Set to `true` to expose Elasticsearch as a route. Set to `false` by defaut.
+
+|`openshift_logging_es_hostname`
+|The external facing hostname to use for the route and the TLS server certificate.
+The default is set to `es`.
```

Comment:
`es.` + openshift_master_default_subdomain.  For example, if `openshift_master_default_subdomain=example.test`, then the default value of `openshift_logging_es_hostname` will be `es.example.test`.